### PR TITLE
added 'rm /trendanalysis/tmp/*' at the beginning of the script

### PR DESCRIPTION
**Pull request by kdelange**

### CODE REVIEW

File: bin/pullRawDataFromDS.sh [hunk 1]
```diff
@@ -259,9 +259,9 @@ then
 	log4Bash 'TRACE' "${LINENO}" "${FUNCNAME:-main}" '0' "this is a testgroup, data should not be removed after 14 days"
 else
 	#
-	# Cleanup old data if data transfer with rsync finished successfully (and hence did not crash this script).
+	# Cleanup old data if data transfer with rsync finished successfully and the rawdata is on prm for atleast 2 days..
```
Comment (pneerincx) [inline bin/pullRawDataFromDS.sh:262]:
```suggestion
	# Cleanup old data if data transfer with rsync finished successfully and the rawdata is on prm for at least 2 days.
```

**Review by pneerincx — DISMISSED** [review]
Just some typos in a comment. Rest looks good!

File: bin/trendAnalyse.sh [hunk 2]
```diff
@@ -84,6 +84,11 @@ function processRawdataToDB() {
 	CHRONQC_TMP="${TMP_TRENDANALYSE_DIR}/tmp/"
 	CHRONQC_DATABASE_NAME="${TMP_TRENDANALYSE_DIR}/database/"
 	PRM_RAWDATA_DIR="${PRM_ROOT_DIR}/rawdata/ngs/${_rawdata}/Info/"
+
+	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "removing files from ${CHRONQC_TMP}"
```
Comment (pneerincx) [inline bin/trendAnalyse.sh:88]:
```suggestion
	log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Removing files from ${CHRONQC_TMP} ..."
```

Comment (kdelange) [reply bin/trendAnalyse.sh:88]: dit zal ik aanpassen

**Pull request merged**